tsconfig.json

```diff
@@ -2,17 +2,19 @@
   "compilerOptions": {
     "outDir": "./dist",
     "sourceMap": true,
-    // "strictNullChecks": true,
+    "strictNullChecks": true,
+    "allowJs": true,
     "noImplicitAny": false,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "module": "CommonJS",
-    "target": "ES2016",
+    "target": "ES2018",
```

Comment:
I thought we are standardizing around `ES2016`?

I guess it depends on our platform. How old can we go? ES2016 would be best for web apps. But since this is consumed by:

1. CLI
2. GUI
3. Mobile

Then perhaps we can go to later versions and get some performance boost?

Reply:
yes I think ES2018 is the best way to go.